setup.py

```diff
@@ -36,7 +36,7 @@
           classifiers=[],
           platforms='any',
           packages=setuptools.find_packages(),
-          package_data={'mnefun': ['run_sss.sh']},
-          scripts=[
-              'bin/simulate_movement.py',
-          ])
+          package_data={'mnefun': ['run_sss.sh', os.path.join('data', 'sss_cal.dat'),
+                                   os.path.join('data', 'ct_sparse.fif')]},
+          scripts=['bin/simulate_movement.py'],
+          requires=['numpy'])
```

Comment:
Please remove the `numpy` requirement here, we don't want `python` trying to use `easy_install` or `pip` or anything else to install it. There are additional requirements anyway, and they match what `mne`'s requirements are.